src/events/SDL_mouse_c.h

```diff
@@ -95,20 +95,19 @@ typedef struct
     SDL_MouseMotionTransformCallback InputTransform;
     void *input_transform_data;
 
+    // integer mode data
+    bool integer_mode_enabled;
+    float integer_mode_residue[4];
```

Comment:
I don't think this isore readable.
What about turning it into a struct?
```suggestion
    struct {
        float xrel_frac;
        float yrel_frac;
        float wheel_x_frac;
        float wheel_y_frac;
    } integer_mode_residue;
```

Reply:
I agree with this feedback. Also, residue is an odd word choice here.